config/config.go

```diff
@@ -325,6 +326,7 @@ var defaultConf = Config{
 	TokenLimit:                   1000,
 	OOMAction:                    "log",
 	MemQuotaQuery:                32 << 30,
+	MemQuotaHashJoin:             32 << 30,
```

Comment:
It can also be controlled by a session variable.
> 1) A new config variable mem-quota-hashjoin in tidb.toml, which is 32GB in default, now takes control of the memory threshold of a hash join. If a hash join exceeds the threshold, it will spill over to the temporary directory. (which could affect performance).
> 2) A new system variable tidb_mem_quota_hashjoin, which is initialized by mem-quota-hashjoin, now takes place in session level as well.